src/main/scala/com/github/vitalsoftware/scalaredox/client/RedoxClient.scala

```diff
@@ -50,7 +56,8 @@ class RedoxClient(
         case Some(info) => Future { info }
         case None       => authorize()
       }
-      response <- execute[T](request.withHttpHeaders("Authorization" -> s"Bearer ${auth.accessToken}"))
+      req = request.addHttpHeaders(play.api.http.HeaderNames.AUTHORIZATION -> s"Bearer ${auth.accessToken}")
```

Comment:
@AndrewZurn This is better programming technique (no hard-coded strings), but only possible by switching from the Play-WS standalone to the full Play-WS package. Ideally, I'd like to keep the external dependencies light on this repo, and will want to switch back to WS-standalone.

Reply:
In other words, good change for now, may revert it when #18 is resolved.

Reply:
Sounds good! Let me switch back to make that change easier for you!